Cargo.toml

```diff
@@ -33,10 +32,11 @@ serde = { version = "^1.0.92", features = ['derive'] }
 serde_json = "^1.0.39"
 # NOTE: keep in sync with wasm-bindgen-test version
```

Comment:
remove `# NOTE: keep in sync with wasm-bindgen-test version` please, it looks that it isn't required anymore

Reply:
yeah... I just want to clear these things after the test passes

Reply:
Ok, let me know, once the PR is ready for code review :)